src/utils/RecordVoiceVisualizer.jsx

```
@@ -73,13 +70,13 @@ export default function RecordVoiceVisualizer() {
     };
     return (
         <div style={{ position: 'relative' }}>
-            <div class="playing">
+            <div className="playing">
                 {Object.entries(renderType).map(([key, value]) => {
                     return <>{renderBar(key, value)}</>;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)